osxkeychain.go

```diff
@@ -359,3 +367,36 @@ func GetAllAccountNames(serviceName string) (accountNames []string, err error) {
 	}
 	return
 }
+
+
+// Add a generic password keychain item with access for the specified application (and ourself).
+// This allows us to create a keychain item and allow access for an additional application or executable.
+func AddGenericPasswordWithApplication(serviceName string, accountName string, data []byte, applicationPath string) (err error) {
```

Comment:
re. the comment above, this should be something like AddKeyDataWithApplication, right?